client/src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,107 @@
+/** @format */
```

Comment:
similar to DEV-015, please structure the Navbar component, like the Card component file structure